BUILD

```diff
@@ -15,7 +15,10 @@
 java_library(
     name = "common-core",
     srcs = glob(["core/src/main/java/com/google/instrumentation/common/*.java"]),
-    deps = ["@jsr305//jar"],
+    deps = [
+        "@guava//jar",
```

Comment:
We have been trying to avoid a dependency on guava in the API. The discussion is in #13.  It looks like you only use it for `@VisibleForTesting` on a non-public method; could you add a comment instead?